### PR TITLE
Wdio5 compatible

### PR DESCRIPTION
**Pull request by jonn-set**
Fixed some dependencies and the tests, the travis build should now pass

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
+  - xvfb
 
 branches:
   only:
     - master
 
 before_script:
   - "export DISPLAY=:99.0"
-  - "sh -e /etc/init.d/xvfb start"
+#   - "sh -e /etc/init.d/xvfb start"
```
Comment (erwinheitzman) [inline .travis.yml]: I would like to see this removed if it's no longer needed

File: package.json [hunk 2]
```diff
@@ -61,7 +66,6 @@
     "gulp": "^4.0.0",
     "gulp-babel": "^8.0.0",
     "gulp-eslint": "5.0.0",
-    "gulp-util": "^3.0.7",
```
Comment (erwinheitzman) [inline package.json:64]: Can this also be removed from the dependencies like you did here or is it required?

File: .travis.yml [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
   - sleep 3 # give xvfb some time to start
```
Comment (erwinheitzman) [inline .travis.yml]: Is this still needed?

File: .travis.yml [hunk 4]
```diff
@@ -1,15 +1,18 @@
 language: node_js
 
 node_js:
-  - '6'
+  - '8.16.2'
```
Comment (erwinheitzman) [inline .travis.yml:4]: Add minimum node version to package.json which is required for v5

**Review by erwinheitzman — DISMISSED** [review]

**Comment by erwinheitzman** [comment]
Hi @jonn-set, thank you for your contribution. I have left some questions and remarks for you.

File: .travis.yml (hunk 1 above)
Comment (jonn-set) [reply .travis.yml]: Doing in the next commit

File: package.json (hunk 2 above)
Comment (jonn-set) [reply package.json:64]: Have done it locally, pushing it now

File: gulpfile.babel.js [hunk 5]
```diff
@@ -1,4 +1,5 @@
-import gutil from 'gulp-util'
+const log = require('fancy-log')
```
Comment (christian-bromann) [inline gulpfile.babel.js]: Can we use `@wdio/logger` for consistency?

**Review by christian-bromann — CHANGES_REQUESTED** [review]
One comment otherwise LGTM

File: gulpfile.babel.js (hunk 5 above)
Comment (jonn-set) [reply gulpfile.babel.js]: Done

**Review by christian-bromann — APPROVED** [review]
👍

**Comment by jonn-set** [comment]
Thanks for releasing it so quickly, appreciate it. Just a small note, I believe `@wdio/mocha-framework` is not a peer dependency, but only a dev dependency. Probably something to fix in the next release :-)

**Pull request merged**